mappings/net/minecraft/entity/mob/MobEntity.mapping

```diff
@@ -3,3 +3,8 @@ CLASS net/minecraft/class_188 net/minecraft/entity/mob/MobEntity
 	METHOD method_634 getTarget ()Lnet/minecraft/class_57;
 	METHOD method_636 setTarget (Lnet/minecraft/class_57;)V
 		ARG 1 target
+	METHOD method_637 attack (Lnet/minecraft/class_57;F)V
+		ARG 1 other
+		ARG 2 distance
+	METHOD method_638 getTargetInRange ()Lnet/minecraft/class_57;
+	METHOD method_640 cantMove ()Z
```

Comment:
What about somethign like `movementBlocked` or `blockMovement` or `preventMovement` ? Feels a bit more descriptive imo.
Btw, field_663 in MobEntity seems to be set to the output of this at the start of tickLiving so field_663 could probably be set to something like `movementBlocked`